src/lib.rs

```diff
@@ -164,7 +171,7 @@ pub fn select_coin_lowestlarger(
         for (idx, input) in sorted_inputs.iter().skip(index) {
             accumulated_value += input.value;
             accumulated_weight += input.weight;
-            estimated_fees = calculate_fee(accumulated_weight, options.target_feerate);
+            estimated_fees = calculate_fee(accumulated_weight, options.target_feerate).unwrap();
```

Comment:
Ideally there should not be any unwraps in business code. All unwraps should be transformed into error propagation via `?`. I think calculate_fee can simple `?` as it returns the `FeeError`. Which should be included in the `SelectionError` enum.

Reply:
Ive fixed this and will push changes once i fix the rest.